docs/getting-started/quick-start/tab-docker/DockerUpdating.md

```diff
@@ -1,35 +1,101 @@
-## Updating
+## Why isn't my Open WebUI updating?
 
-To update your local Docker installation to the latest version, you can either use **Watchtower** or manually update the container.
+To update your local Docker installation of Open WebUI to the latest version available, you can either use **Watchtower** or manually update the container. Follow either of the steps provided below to be guided through updating your existing Open WebUI image.
 
-### Option 1: Using Watchtower
+### Manual Update
 
-With [Watchtower](https://containrrr.dev/watchtower/), you can automate the update process:
+1. **Stop and remove the current container**:
+
+   This will stop the running container and remove it, but it won't delete the data stored in the Docker volume. (Replace `open-webui` with your container's name throughout the updating process if it's different for you.)
+
+```bash
+docker rm -f open-webui
+```
+
+2. **Pull the latest Docker image**:
+
+   This will update the Docker image, but it won't update the running container or its data.
+
+```bash
+docker pull ghcr.io/open-webui/open-webui:main
+```
+
+3. **Remove any existing data in the Docker volume (NOT RECOMMENDED UNLESS ABSOLUTELY NECCESSARY!)**. Skip this step entirely if not needed and move on to the last step:
```

Comment:
We should put this section inside of `:::info` instead of making it a part of an updating instruction